src/07-projects/project-items.apib

```diff
@@ -128,6 +128,19 @@ Get details for a single project item.
             + product (object, nullable)
                 + type: `product` (string)
                 + id: `e2314517-3cab-4aa9-8471-450e73449040` (string)
+            + work_type (object, nullable)
+                + type: `workType` (string)
+                + id: `206bf39b-0161-0058-a015-e530082b0814` (string)
+            + activity_type (object, nullable)
+                + type: `activityType` (string)
+                + id: `ec907500-37d0-0245-8b16-2dbedd0a8ee0` (string)
+            + subject (object, nullable)
```

Comment:
Wondering if this would be better called 'event' instead of 'subject'

Reply:
> Q: Wondering if this would be better called 'event' instead of 'subject'

don't think this applies any longer, as we're returning the info using the same "structure" as TimeTrackings right?